src/renderer/views/User.js

```diff
@@ -1,7 +1,13 @@
 import React, { Component } from 'react';
 import { FaBell } from 'react-icons/fa'
+import VideoWidget from "../components/video/VideoWidget";
+import Image from "react-bootstrap";
```

Comment:
Line `4` should be 
```javascript 
import {Image} from "react-bootstrap"
```

Reply:
Removed some of these import lines as it turns out they weren't used in the final code.